app/views/site/cli.erb

```diff
@@ -40,9 +40,10 @@
         </div>
       <% end %>
 
-      <h2>Want to add more help topics?</h2>
-      <p>Submit a pull request or open an issue with suggestions! The code for this documentation is in <a href="https://github.com/exercism/exercism.io">the exercism.io repository</a> on GitHub.</p>
-      <p>(Check out the <code><a href="https://github.com/exercism/exercism.io/blob/master/x/docs/cli.rb">./x/docs/cli.rb</a></code> and <code><a href="https://github.com/exercism/exercism.io/blob/master/app/views/site/cli.erb">./app/views/site/cli.erb</a></code> files.)</p>
+      <%= erb :"site/_add_more", locals: {
+          presenter: { path: "./x/docs/cli.rb", href: "https://github.com/exercism/exercism.io/blob/master/x/docs/cli.rb" },
+          view: { path: "./app/views/site/cli.erb", href: "https://github.com/exercism/exercism.io/blob/master/app/views/site/cli.erb" }
```

Comment:
perhaps we can just pass `cli` or `help` and then hard code the view and presenter paths with just the path fragment that changes.